src/components/H/H.test.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { renderWithTheme } from '../../../jest/utils';
+import H from '.';
+import Section from '../Section';
+
+describe('H', () => {
+  it('renders as a h2 by default', () => {
+    const { getByText } = renderWithTheme(<H>Hello</H>);
+    expect(getByText('Hello').tagName).toEqual('H2');
+  });
+
+  it('increases levels when within a Section', () => {
+    const { container, getByText } = renderWithTheme(
+      <Section>
+        <H>h3</H>
+      </Section>
+    );
+
+    expect(container.querySelector('h2')).not.toBeInTheDocument();
+    expect(getByText('h3').tagName).toEqual('H3');
+  });
+
+  it('does not render above an h6', () => {
```

Comment:
can we move some of these tests to `components/Section/Section.test.tsx` and `components/Article/Article.test.tsx` accordingly?